dashboard/app/models/foorm/submission.rb

```diff
@@ -43,25 +43,25 @@ def formatted_answers
     parsed_answers = JSON.parse(answers)
 
     # Then, merge in formatted versions of each answer in the submission.
-    parsed_answers.each do |question_id, answer|
-      formatted_answers.merge! get_formatted_answer(question_id, answer)
+    parsed_answers.each do |question_name, answer|
```

Comment:
These changes are unrelated cleanup, to better suggest that the keys in `parsed_answers` are the `name` field for each `question` in a Form.